docs/docs/client-source-without-graphql.md

```diff
@@ -0,0 +1,184 @@
+---
+
+## title: " Client-side sourcing with JSON or YAML"
+
+# Table of Contents
+
+- [Introduction](#Beforehand)
+- [Setup](#Setup)
+- [YAML example](#YAML-example)
+- [JSON example](#JSON-example)
+- [Joining the pieces](#Joining-the-pieces)
+
+# Beforehand
+
+As you come across Gatsby and start discovering the extent of it's possibilities, sometimes you might wonder about the basic things.
+
+Things like importing a JSON file or a YAML file directly into a page.
+
+And that's what you'll build while following this small tutorial.
+
+## Prerequisites
+
+Before we go through the details and code, you should be familiar with the basics of Gatsby.
+
+Check out the [tutorial](https://www.gatsbyjs.org/tutorial/) and brush up on the [documentation](https://www.gatsbyjs.org/docs/).
+In addition to this, some knowledge of [ES6 syntax](https://medium.freecodecamp.org/write-less-do-more-with-javascript-es6-5fd4a8e50ee2) will be useful.
+
+Otherwise just skip this part and move onto the next part.
+
+## Setup
+
+You'll start by creating a new Gatsby website based on the official _hello world starter_.
+
+Open up a terminal and run the following command.
+
+```bash
+gatsby new gatsby-client-sourcing-YAML-JSON https://github.com/gatsbyjs/gatsby-starter-hello-world
+```
+
+After the process is complete, some additional packages are needed.
+
+Change directories to the newly created Gatsby website and issue the following command:
+
+```bash
+npm install --save uuid
+```
+
+Or if Yarn is being used:
+
+```bash
+yarn add uuid
+```
+
+This package is used to ensure uniqueness with React prop keys.
+
+## YAML example
+
+Starting from YAML, if you want to see how to do it using JSON, jump to the [next section](#JSON-example).
+
+### Adding the YAML content
+
+Create a folder called `content` and inside, add a file called `client-data.yml` with the following content inside:
```

Comment:
Later in code it's yaml

```suggestion
Create a folder called `content` and inside, add a file called `client-data.yaml` with the following content inside:
```

Reply:
Nice catch there. Missed the "a" in there